worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -325,6 +332,46 @@ public void butcherBrush(Player player, LocalSession session,
         player.printInfo(TranslatableComponent.of("worldedit.brush.butcher.equip", TextComponent.of((int) radius)));
     }
 
+    @Command(
+        name = "image",
+        desc = "Image brush, uses images as input"
```

Comment:
This does not just "use images as input" as a user would expect, it creates a heightmap from it. This should give some sort of overview of how the brush actually functions, at least in a very general sense.